### PR TITLE
Update kubectl e2e test manifests to apps/v1

### PR DESCRIPTION
**Pull request by janetkuo**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/testing-manifests/kubectl/nginx-deployment1.yaml.in [hunk 1]
```diff
@@ -1,9 +1,12 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1beta2
```
Comment (liggitt) [inline test/e2e/testing-manifests/kubectl/nginx-deployment1.yaml.in]: are any of these used in skewed server tests?

**Comment by dims** [comment]
@janetkuo @liggitt is this ready? (trying to see what needs to be in 1.8 and what can wait)

File: test/e2e/testing-manifests/kubectl/nginx-deployment1.yaml.in (hunk 1 above)
Comment (janetkuo) [reply test/e2e/testing-manifests/kubectl/nginx-deployment1.yaml.in]: kubectl skew tests will test 1.8 kubectl against 1.7 server.

**Comment by janetkuo** [comment]
Moved to 1.9 for kubectl skew tests

**Comment by dims** [comment]
@janetkuo Move this PR to 1.10?

**Comment by kow3ns** [comment]
This is fine to merge, but we can also just wait for 1.10 and update the manifests to apps/v1.

**Comment by janetkuo** [comment]
Updated to apps/v1 and moved to 1.10

**Comment by janetkuo** [comment]
@kow3ns PTAL

**Review by kow3ns — APPROVED** [review]

**Pull request merged**